client/aggregator_test.go

```diff
@@ -2,29 +2,55 @@ package client_test
 
 import (
 	"context"
+	"strings"
 	"testing"
 
 	"github.com/calyptia/api/client"
 	"github.com/calyptia/api/types"
 )
 
+func wantErrMsg(t *testing.T, err error, msg string) {
```

Comment:
Can you move this to `client_test.go` file?
Similar utilities are already there.